pkgs/os-specific/linux/pam_gnupg/default.nix

```
@@ -0,0 +1,24 @@
+{stdenv, fetchFromGitHub, autoconf, automake, libtool, pam, gnupg }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "pam_gnupg";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "cruegge";
+    repo = "pam-gnupg";
+    rev = "fbd75b720877e4cf94e852ce7e2b811feb330bb5";
+    sha256 = "0kqn6xb85jfmhvvbd2lasnci46p2pcwy0wq233za9h7xwfr49f7d";
+  };
+
+  configurePhase = ''
+    mkdir m4
+    ./autogen.sh
+    ./configure --with-moduledir="$out"/lib/security
+  '';
+
+  nativeBuildInputs = [ autoconf automake libtool pam gnupg ];
```

Shouldn't `pam` and `gnupg` be present in `buildInputs`?

Also, in accordance with above:

```suggestion
  nativeBuildInputs = [ autoreconfHook libtool pam gnupg ];
```